### ISSUE TITLE
Area with focus is remembered after the app library is closed and reopened

### ISSUE THREAD

**Issue by leviport** [pop-os/cosmic-applibrary#9]
### Summary: 
If I select the search box, list of apps, or the groups, then close the app library and reopen it, whichever area was selected previously is still selected when I reopen the app library.

### Steps to reproduce:
- Open app library with Super + A
- Single click one of the groups at the bottom
- Close the app library
- Reopen app library with Super + A
- Press left/right arrow on the keyboard to see where the focus is. It should still be in the groups area.

### Expected behavior:
It seems like the text box should have focus by default so that I can start typing to search without having to click it first with my mouse. Then I'd think hitting an arrow key should shift focus to the app list so I can move around and select something from the list with arrows/Enter.

What I'm not sure of is how one would select a group with just the keyboard (maybe hitting Tab?), but that might be out of the scope of this report.

### Version:
```
$ apt policy cosmic-app-library 
cosmic-app-library:
  Installed: 0.1.0~1660784434~22.04~dc85e14
  Candidate: 0.1.0~1660784434~22.04~dc85e14
  Version table:
 *** 0.1.0~1660784434~22.04~dc85e14 1002
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1002 http://apt.pop-os.org/staging/master jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by wash2** [comment]
this should be fixed by bedfe1da354e1b32a1af2df9ed252e5dcaaf376a

**Comment by n3m0-22** [comment]
> It seems like the text box should have focus by default so that I can start typing to search without having to click it first with my mouse.

App library opens to home now by default. Search text box is auto selected and working. 

> Then I'd think hitting an arrow key should shift focus to the app list so I can move around and select something from the list with arrows/Enter.

The arrow keys and enter can be used to select and launch an app. 

> What I'm not sure of is how one would select a group with just the keyboard (maybe hitting Tab?), but that might be out of the scope of this report.

Tab will select the groups only if you are already selecting the last icon in the current group. Otherwise it will cycle through all the apps in the group first. The other option to get to groups faster is to use `Shift` + `Tab` to cycle backwards and then groups can be navigated with the arrow keys. It may be beneficial to allow `Tab` to jump straight to the groups, but I think that would be a new issue.
